contrib/basicauth/.htpasswd

```diff
@@ -0,0 +1 @@
+exampleuser:$apr1$.3BsOjOB$kFcJbNk0nC3/A3vQl5CrL0
```

Comment:
This password file has been generated with the insecure apr1 hash (md5 1000 times, blah blah blah). Here is a [blog](https://www.cryptologie.net/article/126/bruteforce-apr1-hashes/) on how to brute force it.

We should encourage the use of bcrypt, which can be added to htpasswd files with the following command:

```
htpasswd -B file exampleuser
```

This is a small detail, but we should do our part to help the world have better security.

@NathanMcCauley @diogomonica